components/ArticleCard.js

```diff
@@ -36,30 +38,35 @@ export default function ArticleCard({ article, delible, onDelete }) {
       ) : (
         <StyledCard>
           {delible && (
-            <StyledIcon
-              icon="akar-icons:chat-remove"
-              width="25"
+            <StyledFontAwesomeIcon
               onClick={onDelete}
-            />
+              icon={faSquareMinus}
+            ></StyledFontAwesomeIcon>
           )}
-          <img
-            src={urlToImage}
-            alt="article image"
-            width="250"
-            height="150"
-          ></img>
 
-          <article>
+          <StyledArticle>
+            <img
```

Comment:
you can use the ```Image``` Tag, if you want a responsive design

Reply:
Unfortunately I can not use Image here. The images are from everywhere, depending on the article so I dont have control over the domains :(